packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -338,6 +343,67 @@ contract VerifyOPCM is Script {
         return success;
     }
 
+    /// @notice Verifies that the immutable variables in the OPCM contract match expected values.
+    /// @param _opcm The OPCM contract to verify immutable variables for.
+    /// @return True if all immutable variables are verified, false otherwise.
+    function _verifyOpcmImmutableVariables(IOPContractsManager _opcm) internal view returns (bool) {
+        console.log("  Verifying OPCM immutable variables...");
+
+        bool success = true;
+
+        // Get expected addresses from environment variables
+        address expectedSuperchainConfig = Config.expectedSuperchainConfig();
+        address expectedProtocolVersions = Config.expectedProtocolVersions();
+        address expectedSuperchainProxyAdmin = Config.expectedSuperchainProxyAdmin();
+        address expectedUpgradeController = Config.expectedUpgradeController();
```

Comment:
change this to just read directly from env (using [disable forge fmt comments](https://github.com/foundry-rs/forge-std/blob/eb980e1d4f0e8173ec27da77297ae411840c8ccb/src/StdUtils.sol#L72) and semgrep ignore comments)